src/main/java/com/iri/training/repository/ImageRepositoryImpl.java

```diff
@@ -34,30 +34,32 @@ public ImageRepositoryImpl() throws IOException {}
 
 	@Override
 	public Image getImageById(Long imgId) throws SQLException {
-		logger.debug("ENTERED getImageById for id " + imgId);
+		logger.debug("ENTERED getImageById for imgId: " + imgId);
 
 		final Image userImg;
 
 		String sql = property.getString("RETRIEVE_IMG");
 		jdbcTemplate = new JdbcTemplate(dataSource);
 		userImg = jdbcTemplate.query(sql, new Object[]{imgId}, new ImageResultSetExtractor());
-		
-		logger.debug("EXITING getImageById for img " + userImg.toString());
+
+		logger.debug("RETRIEVED Image: " + userImg.toString());
+		logger.debug("EXITING getImageById for imgId: " + imgId);
```

Comment:
As above, no need for retrieved, you can put the Image object in the exiting statement. No need to explicitly call toString() method